business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -191,41 +198,84 @@
 
   <profiles>
     <profile>
-      <id>kie-drools-wb</id>
+      <id>business-central</id>
       <properties>
-        <app.name>kie-drools-wb</app.name>
+        <app.name>kie-wb</app.name>
       </properties>
     </profile>
 
     <profile>
-      <id>kie-wb</id>
+      <id>business-monitoring</id>
       <properties>
         <app.name>kie-wb</app.name>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/ProjectLibraryIntegrationTest.java</exclude>
+              </excludes>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
     </profile>
 
     <profile>
-      <id>kie-wb-monitoring</id>
+      <!--  Enable selenium UI tests to be run with Internet Explorer.
```

Comment:
I removed the ie profile and driver downloader maven plugin config below as it was not used. This PR shouldn't re-introduce those changes.